test/test_events.py

```diff
@@ -253,10 +269,14 @@ def test_UQCS_events_october(uqcsbot: MockUQCSBot):
     messages = uqcsbot.test_messages.get(TEST_CHANNEL_ID, [])
     assert len(messages) == 2
     expected = ("Events in *_October_*\r\n"
-                "*OCT 8 18:00 - 20:00* - `UQCS AGM` - _TBC_\r\n"
-                "*OCT 17 17:00 - 21:00* - `Games Night` - _TBC_\r\n"
-                "*OCT 29 0:00 - OCT 30 23:59* - `Study Session` - _ITLC: GP South 78-217_\r\n"
-                "*OCT 31 0:00 - NOV 1 23:59* - `Study Session` - _ITLC: GP South 78-217_")
+                "*TUE OCT 8 18:00 - 20:00* - `UQCS AGM` - _TBC_\r\n"
+                "*THU OCT 17 17:00 - 21:00* - `Games Night` - _TBC_\r\n"
+                "*TUE OCT 29 0:00 - WED OCT 30 23:59* - `Study Session` - "
+                "_ITLC: "
```

Comment:
Message line continued; begin with `+`.
Move joining characters (` - `) to latter line.
Combine this line with next.